src/ui/router/Browse.js

```diff
@@ -134,7 +134,7 @@ define([
                         .then(children => {
                             let lastChild = children[children.length - 1];
                             if (!lastChild) {
-                                console.debug('Unable to navigate to anything. No root objects found.');
+                                // console.debug('Unable to navigate to anything. No root objects found.');
```

Comment:
this debug message is really annoying, especially in the unit test runs - can i delete it?

Reply:
It really is isn't it. Yes, let's just delete it.